include/dlaf/lapack/tile.h

```diff
@@ -226,6 +226,30 @@ auto hegst(const dlaf::internal::Policy<B>& p, Sender&& s);
 template <Backend B>
 auto hegst(const dlaf::internal::Policy<B>& p);
 
+/// Assemble the Cholesky inverse computing L^H L or U U^H for the triangular tile @a.
+///
+/// Only the upper or lower triangular elements are referenced according to @p uplo.
+///
+/// This overload blocks until completion of the algorithm.
+/// @pre @a is the inverse of the cholesky factor.
```

Comment:
In LAPACK [doc](https://netlib.org/lapack/explore-html/d5/d18/group__lauum_ga109a49941c86e4258f842ceb82952fb1.html) they don't seem to directly refer to Cholesky, but they describe it as a general multiplication of a triangular matrix and its transpose (in-place).

Is there a reason why we might prefer this version of the doc?

Reply:
I limited the scope of the routine to what it is really needed for.
No reason for us to support other cases for now.